src/components/Card/Card.container.ts

```diff
@@ -35,6 +35,52 @@ export const mapStateToProps = (
     {}
   );
 
+  const unstackCard = (oldParentKey: string) => {
+    // Choose a new parent for the card's children
```

Comment:
Auch hier wird die Funktion insgesamt leichter wenn die Root-Card nicht aus dem Stack entfernt werden kann und es muss kein neuer Parent gefunden werden für die verbleibenden Karten

Reply:
Leider wird es nicht so einfach, da es ja intern diese "Baum-Struktur" gibt. Also wenn wir einen Knoten irgendwo in der Mitte unstacken, der aber eigene Kinder hat, wird seine ganze Familie quasi mit ihm ausziehen :D (siehe Bild - ich hoffe man kann da was verstehen) 
<img width="621" alt="baum" src="https://user-images.githubusercontent.com/23505569/89902277-8e9f7f80-dbe6-11ea-984d-a2e157cea2a1.png">
In der nächsten Version von Scrumlr sollten wir den Stack als Componente umsetzen, die alle Karten ohne Hierarchie beinhaltet. Hier mach ich mir aber Sorgen um Kompatibilität, wenn ich es versuche